### PR TITLE
Convert Mail *Attributes interface consumers to jsonable

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

**Comment by blackheaven** [comment]
```
 [ERROR]   RabbitMQClusterTest$ClusterSharing.queuesShouldBeDeclarableOnAnotherNode » Completion
 [ERROR]   RabbitMQClusterTest$ClusterSharing.queuesShouldBeShared » Completion org.testc...
 [ERROR]   RabbitMQClusterTest$ClusterSharing.rabbitMQManagerShouldReturnThreeNodesWhenAskingForStatus » Completion
```

test this please

**Comment by blackheaven** [comment]
test this please

**Comment by chibenwa** [comment]
```
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR] Errors: 
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   MailSpoolTest.sendShouldEnQueueTheMail:57 » IllegalArgument input should at le...
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   MailSpoolTest.sendShouldPositionJMAPRelatedMetadata:69 » IllegalArgument input...
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldCopyMailFromOutboxToSentOnlyOneTimeWhenSuccess:262 » IllegalArgument
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldCopyMailFromOutboxToSentWhenSuccess:143 » IllegalArgument
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldNotMoveMailFromOutboxToSentWhenSuccessIsFalse:177 » IllegalArgument
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldNotMoveMailFromOutboxToSentWhenUsernameNotProvided:209 » IllegalArgument
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldNotThrowWhenMessageIsNotInOutbox:118 » IllegalArgument
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldThrowWhenMailboxException »  Unexpected exc...
[f54bc0b913c7ba99d2591d343490da997336dfd6] [ERROR]   PostDequeueDecoratorTest.doneShouldThrowWhenSentDoesNotExist »  Unexpected exc...
```

File: server/protocols/jmap/src/test/java/org/apache/james/jmap/send/PostDequeueDecoratorTest.java [hunk 1]
```diff
@@ -61,6 +61,7 @@
 import org.apache.mailet.base.test.FakeMail;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
```
Comment (chibenwa) [inline server/protocols/jmap/src/test/java/org/apache/james/jmap/send/PostDequeueDecoratorTest.java]: It is `@Ignore` in junit 4

Comment (blackheaven) [reply server/protocols/jmap/src/test/java/org/apache/james/jmap/send/PostDequeueDecoratorTest.java]: Still JUnit 4? :'(

**Comment by blackheaven** [comment]
```
[ERROR]   RabbitMQClusterTest$ClusterNodesFailure.connectingToAClusterWithAFailedRabbit » Completion
[ERROR]   RabbitMQClusterTest$ClusterSharing.queuesShouldBeDeclarableOnAnotherNode » Completion
[ERROR]   RabbitMQClusterTest$ClusterSharing.queuesShouldBeShared » Completion org.testc...
[ERROR]   RabbitMQClusterTest$ClusterSharing.rabbitMQManagerShouldReturnThreeNodesWhenAskingForStatus » Completion
```

test this please

**Comment by blackheaven** [comment]
```
[ERROR] Errors: 
[ERROR]   CassandraMailRepositoryIntegrationTest.deniedSenderMailShouldBeStoredInCassandraMailRepositoryWhenConfigured:75 » ConditionTimeout
[ERROR] Tests run: 93, Failures: 0, Errors: 1, Skipped: 0
```

Same reproducible error, not knowing what happen

**Comment by trantienduchn** [comment]
```
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR] Errors: 
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   AmqpForwardAttachmentTest.stripAttachmentShouldPutAttachmentsInMailAttributeWhenConfiguredForIt:142 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.base64CalendarAttachmentShouldBePublishedInMQWhenMatchingWorkflowConfiguration:680 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.calendarAttachmentShouldBePublishedInMQWhenMatchingWorkflowConfiguration:587 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.headersShouldBeAddedInMailWhenOneBase64ICalAttachment:660 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.headersShouldBeAddedInMailWhenOneICalAttachment:640 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.headersShouldBeFilledOnlyWithOneICalAttachmentWhenMailHasSeveral:731 » ConditionTimeout
[190912756d07b443ac1626c5c0cbd7b5ac3b1491] [ERROR]   ICSAttachmentWorkflowTest.pipelineShouldSendSeveralJSONOverRabbitMQWhenSeveralAttachments:751 » ConditionTimeout
```

**Pull request closed without merging**